spring-cloud-aws-sns/src/test/java/io/awspring/cloud/sns/integration/SnsTemplateIntegrationTest.java

```diff
@@ -106,4 +116,26 @@ void send_validPersonObject_usesTopicChannel_send_arn_read_sqs() {
 		});
 	}
 
+	@Test
+	void send_test_message_for_existing_topic_name_trigger_trigger_iteration() {
```

Comment:
Lets wrap these two tests into @Nested and the rest also in @Nested to not have setup in @BeforeAll that is not relevant for all the test.